gnosis/eth/account_abstraction/bundler_client.py

```diff
@@ -108,6 +108,7 @@ def _get_user_operation_receipt_payload(
             "id": request_id,
         }
 
+    @cache
```

Comment:
We have other get_chain_id in EthereumClient. Why we are not using it here?
https://github.com/safe-global/safe-eth-py/blob/72b7ed6daded3aa98cd536eb157551365f903a09/gnosis/eth/ethereum_client.py#L1308